optaplanner-benchmark/src/test/resources/org/optaplanner/benchmark/config/testBenchmarkConfig.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<plannerBenchmark>
+<?xml version="1.0" encoding="utf-8"?><plannerBenchmark>
```

Comment:
Unfortunately not; XML transformation, for whatever reason, removes the newline. Thus, a test for exact string comparison fails. As it is just a cosmetic issue and solving it would require more low-level XML handling code, I decided to rather update the test data. I will put there a comment with an explanation.

Reply:
How about https://joel-costigliola.github.io/assertj/core/api/org/assertj/core/internal/Strings.html#assertEqualsIgnoringWhitespace(org.assertj.core.api.AssertionInfo,%20java.lang.CharSequence,%20java.lang.CharSequence)?

Reply:
Yes, I was thinking about that; the downside is we completely lose the check for output formatting. But that could be checked by a separate test.

Reply:
Do we care about output formatting? This transformation is not for humans, right?

Reply:
Not in general; just for the part of config that is supposed to show up in the html report.